### DISCUSSION TITLE
Return an error（empty-ring） using Pyroscope-UI

### DISCUSSION THREAD

**Discussion by WindIsClear** [grafana/pyroscope#3296] [Q&A]
I set up the pyroscope using seven machines, six of which start the pyroscope-distributor,dpyroscope-ingester components, One starts pyroscope-querier,fronte-schedul,query-fronted components,  but returns an error [when](url) I go through pyroscope ui
![image](https://github.com/grafana/pyroscope/assets/44961282/37454469-8622-4df4-a8a6-1121d6e1fed9)

**Comment by WindIsClear** [comment]
this is my config 
![image](https://github.com/grafana/pyroscope/assets/44961282/59818f8b-d830-4a4b-8428-018148c1f1ba)

**Reply by altynbaev** [reply]
I have the same problem, just start store-gateway component and add it to memberlist :)
